audb/core/publish.py

```diff
@@ -262,18 +356,28 @@ def publish(
     r"""Publish database.
 
     A database can have dependencies
-    to files of an older version of itself.
-    E.g. you might add a few new files to an existing database
-    and publish as a new version.
-    :func:`audb.publish` will upload then only the new files
-    and store dependencies on the already published files.
-
-    To allow for dependencies
-    you first have to load the version of the database
+    to media files and tables of an older version.
+    E.g. you might alter an existing table
+    by adding new media files to it
+    and publish it as a new version.
+    :func:`audb.publish` will then upload
+    only the altered table
+    and the new media files,
+    which will be added as new dependencies
+    to the already published files.
```

Comment:
The last two lines are not that clear.
At the beginning we state "A database can have dependencies to media files and tables of an **older version**.",
and here we write that the new media files are added as dependencies instead the old ones.

I guess you wrote it that way to mention that the new files are also put to the dependency list, but I think this is not important for a user. So maybe we state here, that for the already published media files the new version only adds dependencies to them?

Reply:
How about:

![image](https://user-images.githubusercontent.com/10383417/178919785-f04bfb3f-fadf-45eb-9c3e-4e26a6ccbb93.png)

Reply:
or maybe:

![image](https://user-images.githubusercontent.com/10383417/178920008-da43cb0c-56b9-48ad-9e4a-a1f9d62bbe85.png)

Reply:
I would go for your first suggestion. The second one somehow reads like `audb.publish()` would only care about the new and altered files.

Reply:
done